test/test_runner.cc

```diff
@@ -46,24 +45,19 @@ class RuntimeManagingListener : public ::testing::EmptyTestEventListener {
       : runtime_override_(runtime_override), disable_(disable) {}
 
   // On each test start, edit RuntimeFeaturesDefaults with our custom runtime defaults.
+  // The defaults will be restored by TestListener::OnTestEnd.
```

Comment:
I think that this comment is only true for tests that use TestListener (including fuzz tests).

Reply:
which is all tests using this test runner so I think having this comment here makes sense :-)

Reply:
Ah, that makes sense now. Thanks!